src/components/question-builder/components/add-answer.js

```diff
@@ -14,15 +14,13 @@ const AddAnswer = ({
   answerItemCreate,
   disabled
 }) => {
-  const submitHandler = () => {
-    if (answerInputText !== '') answers.push(answerItemCreate(answerInputText));
-  };
   return (
     <form
       onSubmit={e => {
         e.preventDefault();
-        onAnswerSubmit();
-        submitHandler();
+        if (answerInputText !== '') {
```

Comment:
Unnecessary curlys's.
Why is answerItemCreate a seperate func? This should probably just be built into the event handler for submit unless im missing something.

Reply:
answerItemCreate() uses uuid to create a unique key for each item in the array, I would need to import uuid into the actions file to use it. I'd prefer to keep it separate from redux. If you want, I can move the location of that function so it is easy to see what's happening (move it into add-answer.js), what would you prefer?